File: Proxy Pattern/BankAccount.py
class BankAccount:
    def __init__(self,account_owner):
        self.owner= account_owner
        self.balance=0
    def deposit(self,amount):
        self.balance=self.balance+amount
    def withdraw(self,amount):
        self.balance=self.balance-amount

    def printBalance(self):
        print("The balance: ",self.balance)

File: Proxy Pattern/BankAccount_Proxy/BankAccount.java
package BankAccount_Proxy;

public class BankAccount {
    private String owner;//will be directed assigned value to ensure uniqueness of bank owner
    private double balance;

    public BankAccount(String name) {
        owner = name;
        balance=0;
    }

    public void deposit(double amount){
        balance += amount;
    }

    public void withdraw(double amount){
        balance -=amount;
    }

    public double getBalance() {
        return balance;
    }

}
